plugins/rapid7_insightidr/help.md

```diff
@@ -40,6 +47,411 @@ Example input:
+
+Example input:
+
+```
+{
+  "disposition": "BENIGN",
+  "id": "rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|investigation|investigation|True|The investigation for which the disposition was set|
+
+Example output:
+
+```
+{
+  "investigation": {
+    "assignee": {
+      "email": "user@example.com",
+      "name": "Ellen Example"
+    },
+    "created_time": "2018-06-06T16:56:42Z",
+    "disposition": "BENIGN",
+    "first_alert_time": "2018-06-06T16:56:42Z",
+    "last_accessed": "2018-06-06T16:56:42Z",
+    "latest_alert_time": "2018-06-06T16:56:42Z",
+    "organization_id": "174e4f99-2ac7-4481-9301-4d24c34baf06",
+    "priority": "CRITICAL",
+    "rrn": "rrn:example",
+    "source": "ALERT",
+    "status": "OPEN",
+    "title": "Example Title"
+  }
+}
+```
+
+#### Get Investigation
+
+This action allows to get existing investigation by ID or RRN.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|id|string|None|True|The ID or RRN of the investigation|None|rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111|
+
+Example input:
+
+```
+{
+  "id": "rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|investigation|investigation|True|The body of the specified investigation|
+
+Example output:
+
+```
+{
+  "investigation": {
+    "assignee": {
+      "email": "user@example.com",
+      "name": "Ellen Example"
+    },
+    "created_time": "2018-06-06T16:56:42Z",
+    "disposition": "BENIGN",
+    "first_alert_time": "2018-06-06T16:56:42Z",
+    "last_accessed": "2018-06-06T16:56:42Z",
+    "latest_alert_time": "2018-06-06T16:56:42Z",
+    "organization_id": "174e4f99-2ac7-4481-9301-4d24c34baf06",
+    "priority": "CRITICAL",
+    "rrn": "rrn:example",
+    "source": "ALERT",
+    "status": "OPEN",
+    "title": "Example Title"
+  }
+}
+```
+
+#### Update Investigation
+
+This action allows to update existing investigation.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|disposition|string|None|False|Investigation's disposition|['', 'BENIGN', 'MALICIOUS', 'NOT_APPLICABLE']|BENIGN|
+|email|string|None|False|A user's email address for investigation to be assigned|None|user@example.com|
+|id|string|None|True|The identifier of Investigation to be update (ID or RRN)|None|rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111|
+|priority|string|None|False|Investigation's priority|['', 'UNSPECIFIED', 'LOW', 'MEDIUM', 'HIGH', 'CRITICAL']|LOW|
```

Comment:
If empty is a valid priority value for Investigation, it should be included in Set Investigation as well.

Reply:
Empty value in that case means that this parameter won't be updated in the request. In `Set investigation's priority` action, it's required to put a priority value 😃